lib/improver/wxcode/wxcode_decision_tree.py

```diff
@@ -80,8 +80,10 @@ def wxcode_decision_tree():
             'condition_combination': 'OR',
             'diagnostic_fields': ['probability_of_rainfall_rate',
                                   'probability_of_lwe_snowfall_rate'],
-            'diagnostic_thresholds': [AuxCoord(0.03, units='mm hr-1'),
-                                      AuxCoord(0.03, units='mm hr-1')],
+            'diagnostic_thresholds': [AuxCoord(8.33333333e-09,
+                                               units='m s-1'),
+                                      AuxCoord(8.33333333e-09,
+                                               units='m s-1')],
```

Comment:
I thought we were going to convert these units to those of the input cube in weather_symbols.py. This would make it much easier to understand the thresholds required in the decision tree. It is much more like that an incorrect unit will be put in the decision tree (e.g. it is much easier to pick out 1.0 when you meant 0.1 than it is to pick out 2.777778e-07 when you meant 2.777778e-08). Maybe an improvement for the future?

Reply:
I will change the implementation to do this. It will require a rewrite of the check cubes function, which I'll try and get to ASAP.